setup_modules.sh

```diff
@@ -19,6 +19,7 @@ set -eu
 # root directory for the repository
 HPC_STACK_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
+HPC_STACK_VERSION="$(cat ${HPC_STACK_ROOT}/VERSION)"
```

Comment:
use `head -n 1` instead of `cat`.  If for some reason, someone adds description or extra line or whatever, `cat` will give the entire contents of the file.